### PR TITLE
Fix #442: Default language code should be Hindi

### LINKED ISSUE #442: Default audio language to Hindi
We want the default audio language to be Hindi for the prototype.

### PR DESCRIPTION
**Pull request by rt4914**
## How to test this functionality:
Go to any exploration, 
- The default language code in audio-bar should be "HI" , if hindi is present in the language selection list.
- If hindi is not present, then "HI-EN" should be the default language code, if "Hinglish" is present in the language selection list.
- If hindi and hinglish are not present, then "EN" should be the default language code, if "English" is present in the language selection list.
- If all hindi, hinglish and english are not present, then whichever is the first language in the list should be the default language code.

We can see that default selection is hindi in this screenshot.
Screenshot 2019-11-20 at 12 46 19 PM

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by rt4914** [comment]
@BenHenning The priority order of default language code is as follows:
1. Hindi
2. Hinglish
3. English
Else - First available language code.

File: app/src/main/java/org/oppia/app/player/audio/AudioFragmentPresenter.kt [hunk 1]
```diff
@@ -37,12 +37,18 @@ class AudioFragmentPresenter @Inject constructor(
 
   private var prepared = false
   private var selectedLanguageCode: String = ""
-  private var languages = listOf<String>()
+  private var languages = ArrayList<String>()
```
Comment (BenHenning) [inline app/src/main/java/org/oppia/app/player/audio/AudioFragmentPresenter.kt]: Nit: prefer mutableListOf

**Comment by BenHenning** [comment]
Ah, actually to make things easier let's just patch temp-integration-branch-pt2 and change the default to "hi": https://github.com/oppia/oppia-android/pull/358/files#diff-c3bf34b7b6a41d87c0611b9a4b65681bR38.

**Comment by rt4914** [comment]
> Ah, actually to make things easier let's just patch temp-integration-branch-pt2 and change the default to "hi": https://github.com/oppia/oppia-android/pull/358/files#diff-c3bf34b7b6a41d87c0611b9a4b65681bR38.

Closing this PR, since it has been replicated at https://github.com/oppia/oppia-android/pull/446/files

**Pull request closed without merging**